src/testdir/test_goto.vim

```diff
@@ -296,6 +296,7 @@ func Test_gd_string()
       return x;
     }
   [CODE]
+
```

Comment:
This blank line is present in every other test that uses this pattern, so adding it here.